messenger.rst

```diff
@@ -209,6 +220,35 @@ This will also configure the following services for you:
 
         $ composer require symfony/amqp-pack
 
+Redis
+~~~~~
+
+The redis transport will use the redis streams to queue messages.
+
+.. code-block:: bash
+
+    # .env
+    ###> symfony/messenger ###
+    MESSENGER_TRANSPORT_DSN=redis://localhost:6379/messages
+    ###< symfony/messenger ###
+
+This is enough to allow you to route your message to the ``redis`` transport.
+
+If you have multiple systems to receive the same messages you could use different groups
+to achieve this:
+
+.. code-block:: bash
+
+    # .env
+    ###> symfony/messenger ###
+    MESSENGER_TRANSPORT_DSN=redis://localhost:6379/messages/group1/consumer1
+    ###< symfony/messenger ###
+
+.. note::
+
+    In order to use Symfony's built-in Redis transport, you will need the Redis
+    PHP extension (^4.2), a running Redis Server (^5.0) and the Serializer Component.
```

Comment:
s/Server/server/

s/Component/component/

Reply:
Fixed, thank you!